app/models/manageiq/providers/kubevirt/infra_manager/refresh_worker/runner.rb

```diff
@@ -53,13 +53,17 @@ def before_exit(_message, _exit_code)
 
   private
 
+  def provider_class
+    ManageIQ::Providers::Kubevirt
+  end
+
   #
   # Returns the reference to the manager.
   #
   # @return [ManageIQ::Providers::Kubevirt::InfraManager] The manager.
   #
   def manager
-    @manager ||= ManageIQ::Providers::Kubevirt::InfraManager.find(@cfg[:ems_id])
+    @manager ||= Object.const_get("#{provider_class}::InfraManager").find(@cfg[:ems_id])
```

Comment:
since provider_class returns the actual class, you don't need to stringify and then do a const_get, so this can more simply be:

```suggestion
    @manager ||= provider_class::InfraManager.find(@cfg[:ems_id])
```

And then the same in all the other places below